display.py

```diff
@@ -250,6 +250,6 @@
     #if the entire message has moved 'through' the display then
     #start again from the right hand side.
     if offset <= ((len(message)+2)*6) * -1:
-        offset = WIDTH
+        offset == WIDTH
```

Comment:
Use a single equal sign for assignment instead of a double equal sign for comparison. Change `offset == WIDTH` to `offset = WIDTH` to set the value of `offset` correctly.